### ISSUE TITLE
Undefined API error generating report

### ISSUE THREAD

**Issue by jstiops** [AsBuiltReport/AsBuiltReport.Rubrik.CDM#15]
I'm trying to generate a HTML report of a cluster running CDM 5.3 p3. It seems connection to the cluster succeeds and the module starts doing its thing, but failed with "Undefined API" error shown below. Is 5.3 p3 too new for this module perhaps?

command used:
New-AsBuiltReport -Target 'clustername' -Username 'admin' -Password 'password' -Report Rubrik.CDM -Format Html -OutputFolderPath 'C:\reports\rubrik' -Timestamp

Output:
```
Please wait while the Rubrik CDM As Built Report is being generated.
WARNING: [ 08:58:38:161 ] [ Document ] - The table column widths total '99'%. Total column width must equal 100%.
Column widths will be ignored.
New-AsBuiltReport : Undefined API
At line:1 char:1
+ New-AsBuiltReport -Target 'clustername' -Username 'admin' -Password 'password
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (:) [Write-Error], WriteErrorException
    + FullyQualifiedErrorId : Microsoft.PowerShell.Commands.WriteErrorException,New-AsBuiltReport
```

**Comment by tpcarman** [comment]
Not related to the API issue, however the table cable widths warning relates to `Line 130`. When table column widths do not total 100%, the table will be auto-sized rather than the defined column widths.

https://github.com/AsBuiltReport/AsBuiltReport.Rubrik.CDM/blob/5f105deb6f570eebbb760a765477643468df470f/Src/Public/Invoke-AsBuiltReport.Rubrik.CDM.ps1#L130

**Comment by mwpreston** [comment]
Hi @jstiops 
This issue should be solved by ensuring you are running the latest version of the Rubrik PowerShell Module - there were some changes to the API to retrieve users which have been resolved within the latest version (5.3).  Please ensure you have the latest versions of AsBuiltReport.Core, AsBuiltReport.Rubrik.CDM and Rubrik - that should fix your issue.
